util/tmpl.go

```diff
@@ -49,6 +49,10 @@ func MatchTmpl(license string, useSPDXIDs bool) (string, error) {
 		return tmplGPLThreeOrLater, nil
 	case "gpl-3.0-only":
 		return tmplGPLThreeOnly, nil
+	case "AGPL-3.0-or-later":
+		return tmplGPLThreeOrLater, nil
```

Comment:
It should be `tmplAGPLThreeOrLater` here, otherwise the newly added template cannot be matched.